source/opt/inline_pass.cpp

```diff
@@ -266,6 +611,14 @@ bool InlinePass::GenInlineCode(
     return false;
   }
 
+  // Inline instructions of the caller function up to the call instruction.
```

Comment:
Don't call moving instructions in the caller "inline".  Say you are moving them instead.

Reply:
done